### ISSUE TITLE
Battery Draining when no activity is running. 

### ISSUE THREAD

**Issue by TrialMaster** [unfoldedcircle/bug-reports#281]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

Prevent Sleep Is turned on for each activity. Battery was fully drained from 70% overnight with no activity running
![IMG_0055](https://github.com/unfoldedcircle/feature-and-bug-tracker/assets/35364248/821f1929-89c2-435d-839b-aa2a41d821ce)
[UCR2_logs_2024-01-16.txt](https://github.com/unfoldedcircle/feature-and-bug-tracker/files/13956154/UCR2_logs_2024-01-16.txt)

### How to Reproduce

Turn off all activities

Leave remote overnight

### Expected behavior

Prevent Sleep was only expected to do so when the activity is running, not prevent sleep when no activity is running. 

### System version

V1.6.1

### What part of the system affected by the problem?

Core

### Additional context

Turned off running activity at 21:20. HA Notification for battery below 30% triggered at 02:24

**Comment by TrialMaster** [comment]
Turned off running activity 21:20. I have a HA notification for when battery is below 30%, this was triggered at 02:24

**Comment by RomRider** [comment]
Same experience here. Is there anything I can do to help solve this issue? (logs or other...)

**Comment by RomRider** [comment]
```
2024-03-24 12:10:21.691880 +00:00	core	NOTICE	Standby mode is inhibited by: ["Active activity 'Shield'", "Active activity 'Plex'"]
2024-03-24 12:10:20.689762 +00:00	core	NOTICE	Standby mode is inhibited by: ["Active activity 'Shield'", "Active activity 'Plex'"]
2024-03-24 12:10:19.689875 +00:00	core	NOTICE	Standby mode is inhibited by: ["Active activity 'Shield'", "Active activity 'Plex'"]
2024-03-24 12:10:18.689335 +00:00	core	NOTICE	Standby mode is inhibited by: ["Active activity 'Shield'", "Active activity 'Plex'"]
2024-03-24 12:10:17.691388 +00:00	core	NOTICE	Standby mode is inhibited by: ["Active activity 'Shield'", "Active activity 'Plex'"]
2024-03-24 12:10:16.689035 +00:00	core	NOTICE	Standby mode is inhibited by: ["Active activity 'Shield'", "Active activity 'Plex'"]
2024-03-24 12:10:15.689145 +00:00	core	NOTICE	Standby mode is inhibited by: ["Active activity 'Shield'", "Active activity 'Plex'"]
2024-03-24 12:10:14.691992 +00:00	core	NOTICE	Standby mode is inhibited by: ["Active activity 'Shield'", "Active activity 'Plex'"]
2024-03-24 12:10:13.691833 +00:00	core	NOTICE	Standby mode is inhibited by: ["Active activity 'Shield'", "Active activity 'Plex'"]
2024-03-24 12:10:12.691723 +00:00	core	NOTICE	Standby mode is inhibited by: ["Active activity 'Shield'", "Active activity 'Plex'"]
2024-03-24 12:10:11.691830 +00:00	core	NOTICE	Standby mode is inhibited by: ["Active activity 'Shield'", "Active activity 'Plex'"]
2024-03-24 12:10:10.691308 +00:00	core	NOTICE	Standby mode is inhibited by: ["Active activity 'Shield'", "Active activity 'Plex'"]
2024-03-24 12:10:09.689304 +00:00	core	NOTICE	Standby mode is inhibited by: ["Active activity 'Shield'", "Active activity 'Plex'"]
2024-03-24 12:10:08.689805 +00:00	core	NOTICE	Standby mode is inhibited by: ["Active activity 'Shield'", "Active activity 'Plex'"]
2024-03-24 12:10:07.694487 +00:00	core	NOTICE	Standby mode is inhibited by: ["Active activity 'Shield'", "Active activity 'Plex'"]
2024-03-24 12:10:06.689058 +00:00	core	NOTICE	Standby mode is inhibited by: ["Active activity 'Shield'", "Active activity 'Plex'"]
2024-03-24 12:10:05.689661 +00:00	core	NOTICE	Standby mode is inhibited by: ["Active activity 'Shield'", "Active activity 'Plex'"]
2024-03-24 12:10:04.692252 +00:00	core	NOTICE	Standby mode is inhibited by: ["Active activity 'Shield'", "Active activity 'Plex'"]
```

This is what I can see in the log even though those activities are Off (they are in the same activity group and shouldn't even be considered both on at the same time)

**Comment by zehnm** [comment]
Confirmed: the standby inhibitor isn't released if the activity is part of an activity group, and other activities are started as well.

**Comment by zehnm** [comment]
Fixed in next release.
Already available in BETA channel: v1.7.8

**Comment by TrialMaster** [comment]
Thanks so much @zehnm

**Comment by zehnm** [comment]
Fixed in release 1.7.12
